src/start/input_cp2k_motion.F

```diff
@@ -869,16 +869,19 @@ RECURSIVE SUBROUTINE create_geoopt_section(section, location, label, description
       ! collects keywords related to Space Group Symmetry
       CALL keyword_create( &
          keyword, __LOCATION__, name="KEEP_SPACE_GROUP", &
-         description="Keep the space group symmetry operation during optimization.", &
-         usage="KEEP_SPACE_GROUP yes", &
+         description="Detect space group of atoms and preserve during optimization."// &
+         "The space group symmetry is applied to coordinates, forces, and the stress tensor."// &
+         "It does not affect/reduce computational cost."// &
+         "Use EPS_SYMMETRY to adjust the detection threshold.", &
+         usage="KEEP_SPACE_GROUP .TRUE.", &
          default_l_val=.FALSE., lone_keyword_l_val=.TRUE., repeats=.FALSE.)
       CALL section_add_keyword(section, keyword)
       CALL keyword_release(keyword)
 
       ! collects keywords related to precision for finding the space group
       CALL keyword_create( &
          keyword, __LOCATION__, name="EPS_SYMMETRY", &
-         description="Accuracy required for symmetry detection", &
+         description="Accuracy required for the identification of the space group.", &
```

Comment:
Thanks for the clarification!

Looking at the [definition of symprec](https://spglib.github.io/spglib/variable.html#variables-symprec), it seems to me that it actually has a unit - namely the unit of the atomic vectors you pass to spglib.

Are you passing them in in Angstroms?
If yes, then EPS_SYMMETRY should probably get a unit, with Angstrom per default.

Perhaps something like
```suggestion
         description="Accuracy for space group determination. Roughly speaking, two atomic positions v1, v2 are considered identical if |v1 - v2| < EPS_SYMMETRY. ", &
```

I think that should then be enough information for people to figure out by themselves how much precision they have in their structures (i.e. we may not need to mention PDB specifically).

Reply:
@ltalirz EPS_SYMMETRY is dimensionless. It is the difference between the scaled position vectors which is compared to it.

Reply:
Thanks, that is good to know for users - it means that doubling a cell can make atoms equivalent that were not considered equivalent before.

In any case, I think as long as it is documented that's fine.

Reply:
@ltalirz No problem with supercell. Spglib detects that very well. Basically, it adds more translation operations. Rotations are not affected. So atoms that are equivalent in the original cell will also be equivalent with their translation images. I tested this extensively as k-points don't work well (if at all, sorry but there's a problem there in cp2k), and supercells are required.